src/model_metric_reporter.cpp

```diff
@@ -42,171 +42,176 @@ ServableMetricReporter::ServableMetricReporter(const MetricConfig* metricConfig,
         this->buckets.emplace_back(floor(BUCKET_MULTIPLIER * pow(BUCKET_POWER_BASE, i)));
     }
 
-    auto family = registry->createFamily<MetricCounter>("ovms_requests_success",
+    auto familyName = "ovms_requests_success";
```

Comment:
```suggestion
    std::string familyName = "ovms_requests_success";
```
Using `auto` in this context might be std::string or const char*. Probably const char. Why wouldnt we define what type we want?